src/database/resources/api_key.txt

```diff
@@ -0,0 +1 @@
+eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJhdWQiOiIxIiwianRpIjoiNzYyMjcxM2VlYzZkNzFkZWZjYzI1Y2QxNTljMTEwNTY2YzBiYmZiZjZlZjMwMzFiZjQ5OGIwYjUwYzY1ZTVlOTliZTZjMTQ1NzZmYTA4NTUiLCJpYXQiOjE2NDkyNTk0OTEuMzA1NTAyLCJuYmYiOjE2NDkyNTk0OTEuMzA1NTAzLCJleHAiOjQ4MDQ5MzMwOTEuMjkxODk0LCJzdWIiOiI0MTQ1NDg3NSIsInNjb3BlcyI6W119.DHH8AkUVi89PETSWrH92fmvpaZ10F_FtZKMVh7Zeqq_IEy1sEi2-maDAvvnmzJ5mv3b4TEoHirIGRcaPlJssYmnI1lXA4tiT5yheh40rDLvzGWG3nyTagat9fq3eSNYdwRqzNYRbK42khGyrGMVVIwjKccWkbGNHCP2VxDsGTWsHO76N0H5C7GYvWO4Nshm467FzEzXuwOd-Oe5cezcYm0H04XrVh4_WpKFmT1sRDcdS0_cZgTMeisUh-gJ5ZOWO7YH0TCDPiNKmNjKKcCgBw-S1TbRmlvWlmSXiJn1qRZb50q0ZSFTuiW37Y-LFkedRFKG255YorSrivkoVuPVQ9fDd-pE_BUem3dv6-NuA9-9AtuYcnZmCpZvv-c-XJS9pxuRJcGl47n87gL8zqZFRTSuYEfpihiS9nW10u6w6Cup_9cw9ZvICEBV3K5tGssfWE60w48bupfVIRxkeIJhG8sBwClA6A9DIi6YrT7nlUMYoB5SpXvfvdILvPzg_fU-1RKjVW-VqhsjTk1K9Et94dhs_XeQ-NG3yO3VnYGTpBZuCW_-P555RstD23-DAPTrpcazJ0F9xoNVTkZKKD98TAmaF-731gtXEYMcrT8nj2-34Zth9sQAxvBWaeLcJEAuJhmYMLYkiN8GhCPJCNffTfEYEAZ5yc2CKOcCCRwxPmbo
```

Comment:
This API key should not be stored in GitHub. Please remove and send it to me so I can add it to last pass. If you want to keep the file, add the file to the `.gitignore`

Reply:
I would change this line to the python version of a system variable reference. This is how it is done in the build.gradle folder when calling the Google Maps API: `"$System.env.GOOGLE_MAPS_API_KEY"`